### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by beamalsky**
Some styling changes, subject to change after a design review meeting with Derek tomorrow

### CODE REVIEW

File: committeeoversightapp/templates/list.html [hunk 1]
```diff
@@ -5,10 +5,10 @@
 
  <p><a href="/" class="btn btn-success">Create new hearing</a></p>
 
- <table class="table table-striped" id="hearing-table">
+ <table class="table table-striped table-bordered" id="hearing-table">
   <thead>
     <tr>
-      <th scope="col" class="w-75">Hearing Title</th>
+      <th scope="col" id="hearing-title-col">Hearing Title</th>
```
Comment (reginafcompton) [inline committeeoversightapp/templates/list.html]: nice - this works, too!

File: committeeoversightapp/templates/create.html [hunk 2]
```diff
@@ -50,12 +50,9 @@ <h4>Witnesses</h4>
       {% endfor %}
     </div>
 
-    <br />
-
-    <button type="submit" class="btn-lg btn-success float-right">Save hearing</button>
+    <button type="submit" class="btn-lg btn-success float-right" id="save-hearing-button">Save hearing</button>
```
Comment (reginafcompton) [inline committeeoversightapp/templates/create.html]: If you are in front-end mode, you might consider adding a couple Font Awesome icons, particularly for the buttons.

File: committeeoversightapp/templates/list.html (hunk 1 above)
Comment (beamalsky) [reply committeeoversightapp/templates/list.html]: I wanted it more precise than the w-*s allowed! A useful trick for the future, though.

File: committeeoversightapp/templates/create.html (hunk 2 above)
Comment (beamalsky) [reply committeeoversightapp/templates/create.html]: Great pointed, added some

File: committeeoversightapp/templates/list.html [hunk 3]
```diff
@@ -29,6 +29,7 @@
 $(document).ready( function () {
     $('#hearing-table').DataTable({
       "dom": "Bfrtip",
+      "order": [[ 1, "desc" ]]
```
[the remark this answers is not in the document]
Comment (beamalsky) [reply committeeoversightapp/templates/list.html]: The line in `views.py` wasn't working, so I switched it over to here. I didn't do much troubleshooting, though. Is one better than the other?

[the remark this answers is not in the document]
Comment (reginafcompton) [reply committeeoversightapp/templates/list.html]: Ah, I see. Unless you disable ordering (`"order": []`), then dataTables initializes the table with a default order (`[[0, 'asc']]`), which would override the ListView ordering. https://datatables.net/reference/option/order#Default 

I think it makes more sense to order in the dataTables, as you're doing!

**Pull request merged**